src/views/catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/components/UserProvidedInstanceTypeList/UserProvidedInstanceTypeList.tsx

```diff
@@ -19,20 +24,26 @@ import {
 } from '@patternfly/react-core';
 import { TableComposable, TableVariant, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
 
-import UsersInstanceTypeEmptyState from './components/UsersInstanceTypeEmptyState';
+import { isAllNamespaces } from '../../../../../../clusteroverview/OverviewTab/vm-statuses-card/utils/utils';
+
+import UserProvidedInstanceTypesEmptyState from './components/UserProvidedInstanceTypesEmptyState';
 import useInstanceTypeListColumns from './hooks/useInstanceTypeListColumn';
 import { paginationDefaultValues, paginationInitialState } from './utils/constants';
 
-import './UsersInstanceTypeList.scss';
+import './UserProvidedInstanceTypeList.scss';
 
-type UsersInstanceTypesListProps = {
-  userInstanceTypes: V1beta1VirtualMachineInstancetype[];
+type UserProvidedInstanceTypesListProps = {
+  allInstanceTypes: InstanceTypes;
 };
 
-const UsersInstanceTypesList: FC<UsersInstanceTypesListProps> = ({ userInstanceTypes }) => {
+const UserProvidedInstanceTypesList: FC<UserProvidedInstanceTypesListProps> = ({
+  allInstanceTypes,
+}) => {
   const { t } = useKubevirtTranslation();
   const [activeNamespace] = useActiveNamespace();
 
+  const userProvidedInstanceTypes = getUserProvidedInstanceTypes(allInstanceTypes);
```

Comment:
what about sending getUserProvidedInstanceTypes(allInstanceTypes) as the prop ?
<UserProvidedInstanceTypesList userProvidedInstanceTypes={getUserProvidedInstanceTypes(allInstanceTypes)} />

Reply:
Sure. Done!

Reply:
Done!